### ISSUE TITLE
I'm getting this error

### ISSUE THREAD

**Issue by arthurkirkosa** [spatie/laravel-uptime-monitor#58]
After setting `slack`
```php
'notifications' => [
            ...
            \Spatie\UptimeMonitor\Notifications\Notifications\CertificateCheckSucceeded::class => ['slack'],
        ],
```

I get this error
```
[2016-12-18 20:13:39] production.ERROR: ErrorException: Undefined property: Spatie\UptimeMonitor\Notifications\Notifications\CertificateCheckSucceeded::$get in /Users/arthurkirkosa/sites/uptime-monitor/laravel/vendor/spatie/laravel-uptime-monitor/src/Notifications/Notifications/CertificateCheckSucceeded.php:41
```

**Comment by m1guelpf** [comment]
@arthurkirkosa Did you set slack settings in .env?

**Comment by arthurkirkosa** [comment]
Yes I have.

`UPTIME_MONITOR_SLACK_WEBHOOK_URL=https://hooks.slack.com/services/T075ML44D/B3H0APWPR/fKqD8Q79....`

```
'slack' => [
            'webhook_url' => env('UPTIME_MONITOR_SLACK_WEBHOOK_URL'),
        ],
```

**Comment by arthurkirkosa** [comment]
https://github.com/spatie/laravel-uptime-monitor/blob/master/src/Notifications/Notifications/CertificateCheckSucceeded.php#L41 seems to be the line in question;

**Comment by freekmurze** [comment]
Thanks for reporting this. It has been fixed in the newly tagged version `v1.1.2`

Just run `composer update` to pull in the fix.
